technology-overview.md

```diff
@@ -0,0 +1,79 @@
+
+### JavaScript
+
+Agoric works to make [ECMAScript](https://www.ecma-international.org/publications/standards/Ecma-262.htm) (standard defining JavaScript) a suitable language for easily expressing secure smart contracts. Agoric works with ECMA TC39 (which includes browser makers and Node.js representatives) in that direction
+
+Agoric also works with [ECMA TC53](https://www.ecma-international.org/memento/tc53.htm) to make [SES](https://github.com/Agoric/SES) the JavaScript for embedded devices
+
+
+#### Long term
+
+- [Realms](https://github.com/tc39/proposal-realms)
+- [Eventual-send operator](https://github.com/Agoric/proposal-infix-bang)
+- [WeakRefs](https://github.com/tc39/proposal-weakrefs/)
+
+
+#### Short term
+
+- [Realms shim](https://github.com/Agoric/realms-shim)
+- Tooling
+        - [Safe modules](https://github.com/Agoric/safe-modules)
+            - [TOFU (Trust On First Use)](https://github.com/bmeck/tofu/)
+- [Jessie](https://github.com/Agoric/Jessie), a subset of JavaScript that enables easily writing of deterministic smart contracts
+    - [eslint-config-jessie](https://github.com/Agoric/eslint-config-jessie) is an [ESLint](https://eslint.org/) plugin that enforces that some JavaScript files are valid Jessie
+- [SES](https://github.com/Agoric/SES), a secure JavaScript runtime to run untrusted code
+
+
+### Smart Contracts in JavaScript
+
+Agoric works on [ERTP](https://github.com/Agoric/ERTP) (Electronic Rights Transfer Protocol), a framework to express secure smart contracts
+
+ERTP is to writing smart contracts in JavaScript what React is to write user interfaces in plain DOM
+
+ERTP provides well-designed and carefully reviewed smart contract building blocks as well as examples of how they can be used to build complex smart contracts
+
+
+### Blockchains
+
+Various blockchains with various trade-offs exist and are created. Agoric does not develop its own blockchain *consensus* technology, but rather builds on top of existing blockchains
+
+
+#### Long-term
+
+Agoric aims at running its smart contract technology on top of all widely used blockchains.
+
+Agoric will also produce a blockchain using the [Tendermint](https://tendermint.com/docs/introduction/what-is-tendermint.html) consensus integrated with Cosmos and the larger Cosmos ecosystem
+
+
+#### Short-term
+
+Agoric runs on top of blockchains created with the [Cosmos SDK](https://cosmos.network/docs/intro/)
+
+It is planned to enable smart contracts running across different blockchains. To this end, Agoric also works with [Cosmos](https://cosmos.network/) on **IBC (Inter-BlockChain protocol)**
+
+
+### JavaScript on blockchains
+
+*(soon)*
+
+
+## Detailed view
+
+### Vat
+
+A central abstraction to Agoric's technical stack is the **vat**. A vat is a unit of computation running JavaScript
+
+A vat is the equivalent of a Unix process. It has its **own memory space** separate from other vats, and can communicate with other vats via **message passing**
```

Comment:
I think my goal here is more about introducing the term *vat* and a brief description of what it is and how it relates to other concepts developers likely know (object/message passing/(non-)shared memory space), but go no further than that. I would have prefered not introduce this term, but it's so central to the Agoric stack that it's hard to avoid explaining it

I'll add a link to the erights website, but leave it to that if that works with you

In any case, i feel like this content will need to be tested on external devs to figure out what they take off of it and whether it's helped them better understand what Agoric does and whether they are interested in knowing more

Reply:
I agree on all of this. Vats do turn out to be pretty central, and I suspect we're going to have to explain them.

Reply:
Sounds good
I created an issue on this: https://github.com/Agoric/Documentation/issues/6
Also note that i created a "glossary" issue label : https://github.com/Agoric/Documentation/labels/glossary